common/src/main/java/io/pravega/common/io/serialization/VersionedSerializer.java

```diff
@@ -596,21 +596,21 @@ TargetType deserializeContents(InputStream stream) throws IOException {
      * class BaseType { ... }
      *
      * class SubType1 extends BaseType {
-     *     static class SubType1Builder implements ObjectBuilder<SubType1> { ... }
-     *     static class SubType1Serializer extends VersionedSerializer.WithBuilder<SubType1, SubType1Builder> { ... }
+     *     static class SubType1Builder implements ObjectBuilder{@literal <SubType1>} { ... }
+     *     static class SubType1Serializer extends VersionedSerializer.WithBuilder{@literal <SubType1, SubType1Builder>} { ... }
      * }
      *
      * class SubType11 extends SubType1 {
-     *     static class SubType11Builder implements ObjectBuilder<SubType11> { ... }
-     *     static class SubType11Serializer extends VersionedSerializer.WithBuilder<SubType11, SubType11Builder> { ... }
+     *     static class SubType11Builder implements ObjectBuilder{@literal <SubType11>} { ... }
```

Comment:
Same comment as the last comment.

Reply:
replaced @literal with @code. does that make it better?